### PR TITLE
Update bluepysnap version

### PR DESCRIPTION
**Pull request by antonelepfl**
cc: @asanin-epfl 

Based on https://github.com/BlueBrain/snap/compare/v0.5.2...v0.6.1#diff-2eeaed663bd0d25b7e608891384b7298 the only change that I saw was in libsonata

### CODE REVIEW

**Comment by asanin-epfl** [comment]
You need to add `0.1.4` version of libsonata to https://github.com/BlueBrain/spack/blob/develop/var/spack/repos/builtin/packages/py-libsonata/package.py

File: var/spack/repos/builtin/packages/libsonata/package.py [hunk 1]
```diff
@@ -36,6 +37,7 @@ class Libsonata(CMakePackage):
     def cmake_args(self):
         result = [
             '-DEXTLIB_FROM_SUBMODULES=OFF',
+            '-DSONATA_TESTS=OFF',
```
Comment (antonelepfl) [inline var/spack/repos/builtin/packages/libsonata/package.py:40]: I saw this https://github.com/BlueBrain/libsonata/compare/v0.1.3...v0.1.4#diff-2eeaed663bd0d25b7e608891384b7298
Not very sure if I have to add it here though

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/libsonata/package.py:40]: hm, I thought you need to adjust `py-libsonata/package.py`. Check other PRs of bluepysnap and see what files it handled there `libsonata/package.py`, `py-libsonata/package.py` or may be both.

**Comment by asanin-epfl** [comment]
I would wait for Matthias

**Review by matz-e — APPROVED** [review]

**Comment by antonelepfl** [comment]
Is it normal that I still don't see the module?

I just did 

```
[antonel@bbpv1 ~]$ module load unstable
[antonel@bbpv1 ~]$ module av 2>&1 | grep snap
brayns-topologyviewer/0.1.0         py-bluepysnap/0.5.2
```

**Comment by asanin-epfl** [comment]
strange
```
-bash-4.2$ module load unstable
-bash-4.2$ module load py-bluepysnap/0.
py-bluepysnap/0.5.2  py-bluepysnap/0.6.1  
```

**Comment by antonelepfl** [comment]
Now I see them. It took more of what I thought to show up. Thanks

**Pull request merged**